### DISCUSSION TITLE
Reset Configuration does not reset the Audacity Window size to default - now RESOLVED FIXED

### DISCUSSION THREAD

**Discussion by petersampsonaudacity** [audacity/audacity#2765] [Development]
If you use have a non-default size Audacity window and then use **Tools >Reset Configuration** then Audacity does not reset the Audacity Window size to default.

My thinking is that it should as the reset command is a reset to **(mostly)** OOTB factory settings, defaults.

See also related issue #2716 
**Clean up top navigation bar**

Flagging @Tantacrul  as a UX/Design issue

**Comment by LWinterberg** [comment]
Does this really matter though?

**Reply by petersampsonaudacity** [reply]
> Does this really matter though?

That's precisely why I logged this as a "Discussion" rather than a "bug" ...

_If I'd noticed this while @JamesCrook  was writing the **Reset Configuration** the I probably would have asked him to reset the window to default size as part of this._

**Comment by petersampsonaudacity** [comment]
I have added a note about this in the alpha 3,2,0 Manual:
https://alphamanual.audacityteam.org/man/Reset_Configuration#What_is_not_reset

**Comment by petersampsonaudacity** [comment]
On W10 I tested @ksoze95 's branch build:  audacity-win-3.2.0-alpha-20220411+3629298-x64-msvc2022 

Reset Configuration now appears to nicely reset the window size to default in this branch build

**Comment by petersampsonaudacity** [comment]
Testing on W10 with latest alpha master build audacity-win-3.2.0-alpha-20220414+e78705b-x64-msvc2022

This now works properly to resize the window to default size - there remains a minor residual whereby the window is not placed in its default position in the center of the screen.

I have updated the 3.2.0 alpha Manual accordingly:
https://alphamanual.audacityteam.org/man/Reset_Configuration
https://alphamanual.audacityteam.org/man/Tools_Menu#Reset_Configuration
